### ISSUE TITLE
Telekinesis cage bug 

### ISSUE THREAD

**Issue by RFultra** [d3athrow/vgstation13#15588]
If you use telekinesis on a cage and go inside it, when you fly around in space using TK to throw the cage it "hits" you when you change direction or change zlevel

**Comment by Exxion** [comment]
With the outer cover open, I'm guessing
Does this apply to anything else you can be buckled to, like office chairs?

**Comment by RFultra** [comment]
yes with the outer cover open, sorry for not clarifying, I forgot about that feature

**Comment by gbasood** [comment]
i doubt  this will be fixed
